sgx/http.py

```diff
@@ -88,7 +89,10 @@ def send_request(url, method, params, path_to_cert=None):
         response = requests.post(
             url,
             data=json.dumps(call_data),
-            headers=headers, cert=cert, verify=False
+            headers=headers,
+            cert=cert,
+            verify=False,
+            timeout=SERVER_RESPONSE_TIMEOUT
```

Comment:
Suggest to add an option to specify custom timeout for different requests. 
Or rename environment variable to SGX_RESPONSE_TIMEOUT to be able to deduce the meaning of it when using sgx.py in external code.